docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+EOF
+
+versions=( v1.0.0-alpha1 v1.0.0 v1.0.1 v1.1.0 v2.0.0 )
+for version in "${versions[@]}"
+do
+  make generate manifests
+  make docker-build IMG="quay.io/operator-framework/coastal:${version}"
+  mkdir -p manifests
+  kustomize build config/default > manifests/manifests.yaml
+  docker build -t "quay.io/operator-framework/coastal-bundle:${version}" -f plainbundle.Dockerfile .
+  docker push "quay.io/operator-framework/coastal:${version}"
+  docker push "quay.io/operator-framework/coastal-bundle:${version}"
+done
+```
+
+### Create an initial File-Based Catalog (FBC) Image
+Now we need to build and push an initial File-Based Catalog (FBC) image containing
+our bundles. To help highlight the new polling functionality, we are only going to add the `v1.0.0-alpha1`
+bundle to our catalog image in this step and as we go through the rest of the sections we will progressively
+add more of our bundles to the catalog.
+
+- Create a `catalog/` directory
+```sh
+mkdir catalog
+```
+
+- Create a Dockerfile for the FBC image
+```sh
+cat << EOF > catalog.Dockerfile
+FROM scratch
+ADD catalog /configs
+LABEL operators.operatorframework.io.index.configs.v1=/configs
+EOF
+```
+
+- Create the FBC YAML file
+```sh
+cat << EOF > catalog/index.yaml
+{
+  "schema": "olm.package",
+  "name": "coastal"
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0-alpha1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0-alpha1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0-alpha1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "coastal",
+  "entries": [
+    {
+      "name": "coastal.v1.0.0-alpha1"
+    }
+  ]
+}
+EOF
+```
+
+- Build and push the FBC image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+### Create a `Catalog`
+The final preparation step is to create a `Catalog` resource that references the FBC image we built
+in the previous step and specify a polling interval of 15 seconds. This will make sure that the updates
+we make to the catalog image are reflected on our cluster relatively quickly.
+
+Use the following command to create the `Catalog` resource:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: coastal
+spec:
+  source:
+    type: image
+    image:
+      ref: quay.io/operator-framework/coastal-catalog:latest
+      pollInterval: 15s
+EOF
+```
+
+## Only install/upgrade to `v1.0.z` releases
+In this section we are going to create an `Operator` resource that attempts to install our operator
+with a version range of `1.0.x`. This version range ensures we only install z-stream releases for `v1.0`.
```

Comment:
```suggestion
with a version range of `1.0.x`. This version range ensures we only install z-stream releases for `v1.0`, also excluding any pre-release versions.
```
... and then skip [217-219](https://github.com/operator-framework/operator-controller/pull/533/files#diff-242560c892ee9b6e3b4f9ffa1ccedabcab9aae1515aa22985e905087615c8341R217-R219)?

Reply:
[9cecd82](https://github.com/operator-framework/operator-controller/pull/533/commits/9cecd82c89aa82af486c15b987cc97833517ffbf)